nautobot/extras/templates/extras/metadatatype_create.html

```diff
@@ -0,0 +1,91 @@
+{% extends 'generic/object_create.html' %}
+{% load form_helpers %}
+{% load static %}
+
+{% block form %}
+    <div class="panel panel-default">
+        <div class="panel-heading"><strong>Metadata Type</strong></div>
+        <div class="panel-body">
+            {% render_field form.name %}
+            {% render_field form.description %}
+            {% render_field form.data_type %}
+        </div>
+    </div>
+    <div class="panel panel-default">
+        <div class="panel-heading"><strong>Assignment</strong></div>
+        <div class="panel-body">
+            {% render_field form.content_types %}
+        </div>
+    </div>
+    <div class="panel panel-default">
+        <div class="panel-heading"><strong>Choices</strong></div>
+        <div class="panel-body">
+            {% if choices.errors %}
```

Comment:
I'm seeing this show up when there is an error in a different field.

![image](https://github.com/nautobot/nautobot/assets/75227981/56fb926b-78bd-48a1-a7e8-e1e922cd36db)

Reply:
Another copy-paste from custom fields, will investigate.

Reply:
Fixed:

![image](https://github.com/nautobot/nautobot/assets/5603551/862eb1ae-1445-48b4-91da-bb08dd049bc3)